docs/configuration/unit_testing_rules.md

```diff
@@ -77,14 +77,17 @@ series: <string>
 # This uses expanding notation.
 # Expanding notation:
 #     'a+bxc' becomes 'a a+b a+(2*b) a+(3*b) … a+(c*b)'
+#     Read this as series starts at a, then c further samples incrementing by b.
 #     'a-bxc' becomes 'a a-b a-(2*b) a-(3*b) … a-(c*b)'
+#     Read this as series starts at a, then c further samples decrementing by b (or incrementing by negative b).
 # There are special values to indicate missing and stale samples:
 #    '_' represents a missing sample from scrape
 #    'stale' indicates a stale sample
 # Examples:
-#     1. '-2+4x3' becomes '-2 2 6 10'
-#     2. ' 1-2x4' becomes '1 -1 -3 -5 -7'
-#     3. ' 1 _x3 stale' becomes '1 _ _ _ stale'
+#     1. '-2+4x3' becomes '-2 2 6 10' - series starts at -2, then 3 further samples incrementing by 4.
+#     2. ' 1-2x4' becomes '1 -1 -3 -5 -7' - series starts at 1, then 4 further samples decrementing by 2.
+#     3. ' 1x4' becomes '1 1 1 1 1' - shorthand for '1+0x4', series starts at 1, then 4 further samples incrementing by 0.
+#     4. ' 1 _x3 stale' becomes '1 _ _ _ stale' - the missing sample cannot increment, so only 3 samples are produced by the '_x3' expression.
```

Comment:
You mean 2 samples?

Reply:
The `_x3` definitely behaves differently - so the `one missing sample plus 2 more` didn't seem correct. It produces a total of 3 samples.
This is in contrast with `1x3`, which will produce a total of 4 samples due to the shorthand expansion that the numeric version can apply.

Reply:
_x3 should not produce samples

Reply:
Ah - terminology - good point - let me amend that term.